pegasus/sites.v3/hourofcode.com/i18n/public/ru/how-to/afterschool.md

```diff
@@ -46,7 +46,7 @@ We provide a variety of [fun, hour-long tutorials](<%= resolve_url('https://code
 **Покажите им вдохновляющее видео:**
 
   * Оригинальное начальное видео от Code.org, в котором вы сможете увидеть Билла Гейтса, Марка Цукерберга и звезду NBA Криса Боша ([1минутная версия](https://www.youtube.com/watch?v=qYZF6oIZtfc), [5-минутная версия](https://www.youtube.com/watch?v=nKIu9yen5nc) и [9-минутная версия](https://www.youtube.com/watch?v=dU1xS07N-FA))
-  * В[запуске видео Час кода 2013](https://www.youtube.com/watch?v=FC5FbmsH4fw) или <% if @country == 'uk' %>[Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc)<% еще %>[Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc) <% end %>
+  * В[запуске видео Час кода 2013](https://www.youtube.com/watch?v=FC5FbmsH4fw) или <% if @country == 'uk' %>[Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc) <% else %> [Час кода 2015 видео](https://www.youtube.com/watch?v=7L97YMYqLHc) <% end %>
```

Comment:
So the spaces outside the brackets get stripped? How does one indicate that starting or ending whitespace shouldn't be stripped?